app/assets/javascripts/admin/payments/services/payments.js.coffee

```diff
@@ -0,0 +1,11 @@
+# Darkswarm.factory 'Payments', ()->
```

Comment:
This looks like a temporary commit. @stveep Can you rebase your commit history so that each commit makes sense on its own? That makes reviewing a lot easier.